tests/fixtures/ecma-version/6/modules/import-named-as-specifier-keyword.result.js

```diff
@@ -0,0 +1,275 @@
+module.exports = {
```

Comment:
Might want to add Tern-JS copyright at the top, since this is copied from their repository